changes/ce/feat-12089.en.md

```diff
@@ -0,0 +1,10 @@
+Added a technical preview of the new persistent session implementation based on RocksDB.
+Please note that this feature is in alpha stage and must not be enabled in the production systems.
+
+Features missing in the early preview version of the new persistent session implementation:
+
+- Shard failover
+- Retained messages
+- Will message handling
+- Shared subscriptions
+- Subscription IDs
```

Comment:
This one should be covered once  #12055 is merged.

Reply:
This item can be removed once this PR is merged.